scss/_patterns_accordion.scss

```diff
@@ -78,22 +77,21 @@
   }
 
   // stylelint-disable selector-max-type
-  h2.p-accordion__heading > .p-accordion__tab::before,
-  .p-heading--2 > .p-accordion__tab::before {
-    $icon-top-position: calc(#{$table-cell-vertical-padding + (map-get($line-heights, h2) - $icon-size) * 0.5});
-    top: $icon-top-position;
+  .p-heading--2 > .p-accordion__tab::before,
+  h2.p-accordion__heading > .p-accordion__tab::before {
+    @include vf-icon-size($x-height);
+    vertical-align: 0;
   }
 
-  h3.p-accordion__heading > .p-accordion__tab::before,
-  .p-heading--3 > .p-accordion__tab::before {
-    $icon-top-position: calc(#{$table-cell-vertical-padding + (map-get($line-heights, h3) - $icon-size) * 0.5});
-    top: $icon-top-position;
+  .p-heading--3 > .p-accordion__tab::before,
+  h3.p-accordion__heading > .p-accordion__tab::before {
+    @include vf-icon-size($default-icon-size);
```

Comment:
Is this needed when it's default icon size? Or is the size different in standard accordion?

Reply:
I was about to remove it but I thought maybe it was to make sure that the icon next to an `h3` have the right size no matter the base size. But you're right that in this case it's unnecessary. I just deleted it.